src/SFA.DAS.Campaign.Web/Views/Shared/Sidebar/_employer.cshtml

```diff
@@ -0,0 +1,22 @@
+﻿<ol class="sidebar-nav">
+    <li class="sidebar-nav__section sidebar-nav__section--active">
+        <a href=""> How much is it going to me?</a>
+        <ul>
+            <li><a href=""> Paying for apprentices using a transfer of funds</a></li>
+            <li><a href=""> Transfer funds from a levy payer</a></li>
+            <li><a href=""> Paying the levy</a></li>
+        </ul>
+    </li>
+    <li class="sidebar-nav__section">
+        <a href=""> Choose an apprenticeship</a>
+        <ul>
+            <li><a href=""> Paying for apprentices using a transfer of funds</a></li>
+            <li><a href=""> Transfer funds from a levy payer</a></li>
+            <li><a href=""> Paying the levy</a></li>
+        </ul>
+    </li>
+    <li class="sidebar-nav__section"><a href=""> Choose a training provider</a></li>
+    <li class="sidebar-nav__section"><a href=""> Hire an apprentice</a> </li>
+    <li class="sidebar-nav__section"><a href=""> Monitor progress</a></li>
+    <li class="sidebar-nav__section"><a href=""> Assessment &amp; qualification</a></li>
+</ol>
```

Comment:
I know we are trying to get a view of what it looks like but I wouldn't add anything that didn't have an actual link on

Reply:
this is resolved in another PR